Dockerfile

```diff
@@ -1,18 +1,22 @@
-FROM phusion/baseimage:0.9.18
+FROM python:3.6-slim-stretch
 MAINTAINER Holger Schinzel <holger@dash.org>
 
-ARG USER_ID
-ARG GROUP_ID
+RUN apt-get update && \
+    apt-get install git curl mysql-client cython3 -y && \
```

Comment:
For the "official" image, do we want to ship it w/MySQL client libs also? We can, but AFAIK mostly just Holger uses this (but I could be wrong).

Reply:
@schinzelh What do you think? What are the reasons to favor MySQL with Sentinel?